pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -94,7 +94,10 @@ sub _suggest_kernelcare {
     my ($self) = @_;
 
     # Abort if the system won't benefit from KernelCare.
-    return if !Cpanel::KernelCare::system_supports_kernelcare() or Cpanel::Security::Advisor::Assessors::Symlinks->new->has_cpanel_hardened_kernel();
+    return if !Cpanel::KernelCare::system_supports_kernelcare() || Cpanel::Security::Advisor::Assessors::Symlinks->new->has_cpanel_hardened_kernel();
+
+    # Abort if kernelcare is already licensend
+    return if try { Cpanel::KernelCare::Availability::system_license_from_cpanel(); };
```

Comment:
This is indeed available in 70, so it works with our earliest supported version.